drivers/include/mma7660.h

```diff
@@ -0,0 +1,238 @@
+/*
+* Copyright (C) 2016 University of California, Berkeley
+*
+* This file is subject to the terms and conditions of the GNU Lesser
+* General Public License v2.1. See the file LICENSE in the top level
+* directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_mma7660 MMA7660 Accelerometer
+ * @ingroup     drivers_sensors
+ * @brief       Driver for the Freescale MMA7660 3-Axis accelerometer.
+ *              This driver only implements basic functionality.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the MMA7660 accelerometer driver.
+ *
+ * @author      Michael Andersen <m.andersen@cs.berkeley.edu>
+ */
+
+#ifndef MMA7660_H
+#define MMA7660_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+  * @brief   Device descriptor for an MMA7660 device
+  * @{
+  */
+typedef struct
+{
+  i2c_t i2c;      /**< the I2C handle */
+  uint8_t addr;   /**< the device I2C address */
+} mma7660_t;
```

Comment:
I think this should rather be:

```
typedef struct {
    mma7660_params_t params;
} mma7660_t;
```

and consequently defined after `mma7660_params_t`. First, both attributes are part of params (so duplicated here), and second you may want to and access the params somewhere, see other drivers like `mag3110`, too.